app/routes/admin/content/delete.tsx

```diff
@@ -35,6 +35,9 @@ export async function action({ request }: ActionArgs) {
 const DeleteContent = () => {
   const { content } = useLoaderData<typeof loader>();
   const [filteredContent, setFilteredContent] = useState(content);
+  const [selectedRow, setSelectedRow] = useState("");
```

Comment:
Foreslår å rename denne slik at det blir litt enklere å skjønne hvorfor vi trenger den 😊 
```suggestion
  const [itemToDelete, setItemToDelete] = useState("");
```